ui/static/ui/js/learning_resources.jsx

```diff
@@ -6,73 +6,156 @@ define('learning_resources', [
   var StatusBox = Utils.StatusBox;
   var Select2 = Utils.Select2;
 
-  var VocabularyOption = React.createClass({
+  var TermListItem = React.createClass({
     render: function () {
-      var options = _.map(this.props.terms, function (term) {
+      return (
+        <li className="applied-term list-group-item">
+          <strong>{this.props.label}: </strong> {this.props.terms}
+        </li>
+      );
+    }
+  });
+
+  var VocabSelect2 = React.createClass({
+
+    render: function () {
+
```

Comment:
Yep -- there were comments in there, now gone -- will tighten up the whitespace. Thx.